src/aiodynamo/errors.py

```diff
@@ -1,10 +1,165 @@
-class ItemNotFound(Exception):
+import json
+
+
+class AIODynamoError(Exception):
+    pass
+
+
+class EmptyItem(AIODynamoError):
+    pass
+
+
+class ItemNotFound(AIODynamoError):
     pass
 
 
-class TableNotFound(Exception):
+class UnknownError(AIODynamoError):
+    def __init__(self, status: int, body: bytes):
+        self.status = status
+        self.body = body
+        super().__init__(body)
```

Comment:
It would be nice to offer custom `__str__` and/or `__repr__` for this exception.

Reply:
Eh, now I see that it inherits stringy methods from Exception, so `body` will be reported, but not status...
Maybe that's enough 🤔